testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -2180,6 +2180,35 @@ QUnit.module('keyboard navigation', {
         fx.off = false;
     }
 }, () => {
+    QUnit.test('navigation keys should focus items', function(assert) {
```

Comment:
1. I don't see a reason to write several asserts for the single key. 
2. This test name is too common for me: home/end, arrowLeft/arrowRight are "navigation keys" too. This is common because this test is not atomic. Let's create 2 tests: "up/down key press should move tag focus" and "pagedown/pageup key press should move tag focus"

Reply:
yep, thought about it